boards/airfy-beacon/doc.txt

```diff
@@ -0,0 +1,209 @@
+/**
+ * @defgroup    boards_airfy-beacon Airfy Beacon
+ * @ingroup     boards
+ * @brief       Support for the Airfy Beacon board
+ *
+ * ## Overview
+ * 
+ * The Airfy Beacon is utilizing a Nordics NRF51822QFAA SoC.
+ * The SoC features 16Kb of RAM, 256Kb of flash ROM and comes on top of the usual micro-controller peripherals with a 2.4GHz radio that supports both Nordics proprietary ShockBurst as well as Bluetooth Low Energy (BLE).

```

Comment:
It would be great to split the line so that they fit in the 80 characters maximum length. Do you think it's possible to adapt your script for this ?

Reply:
yes, I think so